usagov/usagov-2021
# Usagov 346 remove mail safety module

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-346

## Description

Removed the Mail Safety Module.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


1. Run `bin/init` and `bin/drush cim`
2. Create a basic page and save it as a draft to make sure that the content workflow still works
3. Change the status to "Needs Review" and then to "Published". 
4. Go to the modules page and check that the "Mail Safety" module it's not there

### Requires New Config

### Requires New Content

### Validation Steps

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
6. Find the commit of this pull request.
7. Build and deploy the changes.
8. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
